lib/builderator/tasks/packer.rb

```diff
@@ -137,31 +137,38 @@ def remote_tag(profile)
         allowed_cred_keys = %w(access_key_id secret_access_key session_token)
 
         images.each do |image_name, (image, build)|
-          filters = [{
-            :name => 'name',
-            :values => [image_name]
-          }]
-
-          if build.tagging_role.nil?
-            say_status :complete, 'No remote tagging to be performed as no IAM role is defined'
-            return
-          end
+          ami_regions = build.ami_regions
+          ami_regions << Config.aws.region
+          ami_regions.uniq!
+          ami_regions.each do |region|
+            filters = [{
```

Comment:
Not sure it's really necessary as it is working, but I will to keep things consistent.